src/commcare_cloud/ansible/roles/java/tasks/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - name: installing repo for Java 8+ in Ubuntu
   apt_repository: repo='ppa:openjdk-r/ppa'
+  when: ansible_distribution_version == '14.04'
```

Comment:
openjdk-8 is already in bionic https://packages.ubuntu.com/bionic/openjdk-8-jdk so I think this should just be removed

Reply:
You mean it should be removed once we're officially dropping trusty? I agree with that